application-services/application-services.env

```diff
@@ -0,0 +1 @@
+APPLICATION_SERVICES_BUILD_ID=124.20240126050308
```

Comment:
Yes yes yes this is so much easier than tracking it down in two different files 🙌